eclipse-collections-code-generator/src/main/resources/test/map/mutable/abstractMutablePrimitivePrimitiveMapTestCase.stg

```diff
@@ -27,17 +27,19 @@ import java.util.NoSuchElementException;
 import org.eclipse.collections.api.block.function.primitive.<name1>To<name2>Function;
 import org.eclipse.collections.api.block.function.primitive.<name2>Function;
 import org.eclipse.collections.api.block.function.primitive.<name2>Function0;
-<if(!sameTwoPrimitives)>import org.eclipse.collections.api.block.function.primitive.<name2>To<name2>Function;<endif>
```

Comment:
any particular reason why the imports are rearranged? It causes unnecessary git churn and confusion.

Reply:
Sorry, no specific reason other than having the imports in alphabetical order. Should be fine now